.travis.yml

```diff
@@ -1,89 +1,87 @@
-sudo: required
+# sudo: required
 branches:
     only:
         - master
-        - potato-1490727
-cache:
-    pip: True
-    directories:
-      - node_modules
-      - downloads
 language: python
+python:
+    - "2.7"
 services:
     - docker
-    - mysql
-    - redis
+    # - mysql
+    # - redis
 env:
     global:
-        - DATABASE_URL=mysql://root:@127.0.0.1:3306/kuma
-        - DJANGO_SETTINGS_MODULE=kuma.settings.travis
+#         - DATABASE_URL=mysql://root:@127.0.0.1:3306/kuma
+#         - DJANGO_SETTINGS_MODULE=kuma.settings.travis
         - DOCKER_COMPOSE_VERSION=1.23.2
```

Comment:
We just need this also within the `env: global:` section, which solves the permission issue:
```
- UID=0
```